src/main/java/com/fasterxml/jackson/databind/ser/BeanPropertyWriter.java

```diff
@@ -24,6 +24,8 @@
 import com.fasterxml.jackson.databind.util.ClassUtil;
 import com.fasterxml.jackson.databind.util.NameTransformer;
 
+import static com.fasterxml.jackson.databind.ser.impl.PropertySerializerMap.emptyForProperties;
```

Comment:
Let's revert this change and below back to how it was.
This is not related to the PR.

```java
_dynamicSerializers = (ser == null) ? emptyForProperties() : null;
```

Reply:
done, though it was a clean way to import statically